web/src/api/auth/graphql/insert-membership.graphql

```diff
@@ -0,0 +1,26 @@
+mutation InsertMembership(
```

Comment:
given we're inserting the user here as well might be worth renaming it for better clarity. also, maybe best to hardcode the `MEMBER` role from here?